app/clients/telemetry.js

```diff
@@ -12,21 +13,46 @@ const pool = mysql.createPool({
 
 const query = util.promisify(pool.query).bind(pool);
 
-const getLatestRecord = async (buoyId, tableType) => {
+const ALL_COLUMNS = ["*"];
+const getColumns = (variables = ALL_COLUMNS) => {
+  // note - this is okay because this code is the only source of variable names - this would
+  // be bad if we allowed user defined variables here
+  return (columns = variables.map((variable) => SqlString.raw(variable)));
+};
```

Comment:
Took me a while to trace that out, but it looks like all the vars are being set through the CORE_METRICS, is that right?

Reply:
Either there or the `ALL_COLUMNS` default.  I could see a world where we have some other pre-canned sets of tables/variables, but always defined from within the api.